checks/documentation.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2020 Security Scorecard Authors
```

Comment:
Change filename -> check_documentation.go

Reply:
It is already within the `checks` package.  Adding prefix doesn't add any value and also `_` aren't common within standard `go` naming though there are few exceptions.